src/common/buffer.cc

```diff
@@ -125,7 +129,7 @@ static simple_spinlock_t buffer_debug_lock = SIMPLE_SPINLOCK_INITIALIZER;
     if (r < 0)
       return r;
     buf[r] = '\0';
-    size_t size = strict_strtol(buf, 10, &err);
+    size_t size = strtol(buf, nullptr, 10);
```

Comment:
The main reason for me to discontinue use strict_strtol is that entities read from linux /proc tend to be followed by newline. strict_strtol just didnt work. 
I guess it could be fixed by "if (buf[r-1] =='\n') buf[r-1]=0;
Which way should I go:
1) Fiddle with buffer contents
2) switch from strict_strtol to strtol ?